app/uk/gov/hmrc/apisubscriptionfields/model/JsonFormatters.scala

```diff
@@ -36,16 +37,42 @@ trait SharedJsonFormatters {
 
 trait JsonFormatters extends SharedJsonFormatters {
 
+  implicit val ValidationRuleTypeReads = Reads.enumNameReads(ValidationRuleType)
+
+  val validationRuleReads: Reads[ValidationRule] = (
+    ((JsPath \ "validationRuleType").read[ValidationRuleType]) and
+      ((JsPath \ "value").read[String] or Reads.pure(""))
+  )(ValidationRule.apply _)
+  val validationRuleWrites = Json.writes[ValidationRule]
```

Comment:
Too many parenthesis here. This would be fine.

```
val validationRuleReads: Reads[ValidationRule] = (
    (JsPath \ "validationRuleType").read[ValidationRuleType] and
    ((JsPath \ "value").read[String] or Reads.pure(""))
  )(ValidationRule.apply _)
```

Reply:
Done